dev-java/openjdk/openjdk-11.0.8_p10.ebuild

```diff
@@ -131,6 +138,14 @@ pkg_setup() {
 
 src_prepare() {
 	default
+
+
+	# remove not compilable module (hotspot jdk.hotspot.agent)
+	# this needs libthread_db which is only provided by glibc
+	#
+	# haven't found any way to disable this module so just remove it.
+        rm -rf "${S}"/src/jdk.hotspot.agent
```

Comment:
maybe guard it by `use elibc_musl && { rm "${S}"/src/jdk.hotspot.agent || die ; }`
note the missing `die`
also use tabs.